wait_for_db.sh

```
@@ -0,0 +1,13 @@
+set -e
```

Add a shebang at the top of the file, e.g. #!/usr/bin/env bash. Without it the script may not run when executed directly. This helps portability and clarity. Also keep set -e to fail fast.